reporter/resources/boundary/boundary.license

```diff
@@ -0,0 +1,12 @@
+OpenStreetMap is open data, licensed under the Open Data Commons Open
```

Comment:
We should probably just make one copy of the license and then copy it over as needed (not a blocker for this pr)